### DISCUSSION TITLE
Comments not highlighting correctly in language

### DISCUSSION THREAD

**Discussion by oWretch** [zed-industries/zed#22278] [Feature Requests]
I've created the basics of an extension to support Bicep. I have the language server working and most of the syntax highlighting. My only issue is that comments are not highlighted as they ought. The are just ignored in the highlighting. I don't see any errors in the log so am at a loss as to why.

The repo with the extension is here: https://github.com/oWretch/zed-extension-bicep

**Comment by WhySoBad** [comment]
Hi, I'm having the same issue. Did you manage to resolve it?

**Reply by oWretch** [reply]
I did thanks to !no ones uncle in Discord. My issue was that the tree sitter config I copied had `@spell` on the comment definition which caused the problem. Removing it made comments highlight immediately. https://github.com/oWretch/zed-extension-bicep/commit/2149874e747881950dd650cc94c8e04788e5150d

**Reply by WhySoBad** [reply]
Thanks a lot for your quick reply!
Turns out my `highlights.scm` also had `(comment) @comment @spell` and removing the `@spell` part solved the problem for me as well.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
